### ISSUE TITLE
Next feature release

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#6907]
In order of my(our) preference to have faster release cycles from `master` I propose to make another merge into `release` rather soon next days and publish a RC-version.

Especially I am thinking of the work in progress regarding #6519, which should go into `master` at a point of time which allows enough time for debugging. So after merging to `release` the corresponding PR could be merged into `master`.

There are a few other things, which IMHO need to be done before we can merge to `release`:
- [x] PR #6906 
- [x] Upload strings to Crowdin (we have some new prompts to be translated since last release)
- [x] Fix #6903
- [x] Wait for incoming translations
- [x] Download incoming translations
- [x] Merge branches

Besides these, are there any thoughts about the further way forward regarding #6594 ?

Your opinions?

**Comment by pstorch** [comment]
I think a fix for #6903 is worth adding. I want to have a look at it tonight.

Regarding #6594: I use the BetaMap as default for a long time now and I don't miss anything important. So, a :+1: from my side.

**Comment by Lineflyer** [comment]
> Regarding #6594: I use the BetaMap as default for a long time now and I don't miss anything important. So, a 👍 from my side.

I mainly agree, so we "just" have to find someone to bring it out of the "beta selection state" in a first step as discussed in #6594. @rsudev ;)

I added Fix for #6903 above

**Comment by Lineflyer** [comment]
Who can upload to Crowdin?

**Comment by rsudev** [comment]
I can do that

**Comment by rsudev** [comment]
Done.

**Comment by Lineflyer** [comment]
Thanks, I added three new items (wait for translations, import, merge branches) to the ToDo list above.

**Comment by rsudev** [comment]
The bulk of translation changes we are likely to receive is merged.
If we merge to release, we still can import additional incoming translations there.

**Comment by Lineflyer** [comment]
Cool..
Merge.? Merge! Merge! Merge!

**Comment by pstorch** [comment]
Merge to release done. This time with changelog updates :wink:

**Comment by Lineflyer** [comment]
Thanks, I will check tonight and generate a new RC/Beta.

**Comment by Lineflyer** [comment]
A new RC version (based on `release`) has been published on Google Play as beta version.

**Comment by rsudev** [comment]
Today the translation to Turkish was completed (the fastest translation ever I'd say).
I think it would be good to include it.

**Comment by Lineflyer** [comment]
Wow. Just some days ago we have been asked on FB and I enabled it.
Sure we should include it.

**Comment by rsudev** [comment]
I put it to the nightly, if nothing comes after a smoke test, I can put it to release as well.

**Comment by pstorch** [comment]
Should we include #6917?

**Comment by Lineflyer** [comment]
As #6917 has been included, I compiled a new RC version 2018.01.20-RC.
Will briefly check on regressions regarding login procedures and publish it to our beta channel afterwards.

**Comment by Lineflyer** [comment]
The RC seems to perform good, no relevant support mail, no suspicious new crash reports on Play.

However I personally uggest waiting with the release for two more weeks basically onle due to my upcoming vacation (until 13.02) where I won't be able to take care of support mail and social media channels.

Anyway if something happens which requires to release, the RC version is a good basis but someone else needs to take care about that.

**Comment by Lineflyer** [comment]
Done meanwhile, refer to #6929
